src/sssom/util.py

```diff
@@ -105,6 +105,59 @@ def with_converter(
             metadata=metadata or get_default_metadata(),
         )
 
+    @classmethod
+    def from_mappings(
+        cls,
+        mappings: List[SSSOM_Mapping],
+        *,
+        converter: HINT = None,
+        metadata: Optional[MetadataType] = None,
+    ) -> "MappingSetDataFrame":
+        """Instantiate from a list of mappings, mapping set metadata, and an optional converter."""
+        metadata = dict(ChainMap(metadata or {}, get_default_metadata()))
+        mapping_set = MappingSet(mappings=mappings, **metadata)
+        return cls.from_mapping_set(mapping_set=mapping_set, converter=converter)
+
+    @classmethod
+    def from_mapping_set(
+        cls, mapping_set: MappingSet, *, converter: HINT = None
+    ) -> "MappingSetDataFrame":
+        """Instantiate from a mapping set and an optional converter.
+
+        :param mapping_set: A mapping set
+        :param converter: A prefix map or pre-instantiated converter. If none given, uses a default
+            prefix map derived from the Bioregistry.
+        :returns: A mapping set dataframe
+        """
+        doc = MappingSetDocument(converter=ensure_converter(converter), mapping_set=mapping_set)
+        return cls.from_mapping_set_document(doc)
+
+    @classmethod
+    def from_mapping_set_document(cls, doc: MappingSetDocument) -> "MappingSetDataFrame":
+        """Instantiate from a mapping set document."""
+        if doc.mapping_set.mappings is None:
+            return cls(df=pd.DataFrame(), converter=doc.converter)
```

Comment:
what is this cls?

Reply:
Maybe some strange magic that refers to the parent class.

Reply:
Class methods allow access to the class. So if you want to make an instance of the class from a classmethod, you can refer to `cls` instead of writing the name of the class. This makes it more impervious to tricky inheritance.

Reply:
Cool!